app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,92 @@
+require 'sinatra/base'
+require 'rack-flash'
+class TweetController < ApplicationController
+
+    enable :sessions
+    enable :method_override
+
+    use Rack::Flash
+ 
+    get '/tweets' do
+        if logged_in?(session)
+            @user = current_user(session)
+            @tweets = Tweet.all 
+            @users = User.all
+        else
+            redirect to "/login"
+        end
+        erb :'tweets/tweets'
+    end
+
+    get '/tweets/new' do
+        if logged_in?(session)
+            erb :'tweets/create_tweet'
+        else
+            redirect to "/login"
+        end
+    end
+
+    post '/tweets' do
+        @tweet = Tweet.new(params)
+
+        if @tweet.content == ""
```

Comment:
I think you could use [ActiveRecord validations](http://guides.rubyonrails.org/active_record_validations.html) (e.g. `validates_presence_of :username`) to perform these types of checks more easily.